src/obis-observer/commands/downlink/AddMeterProfile.ts

```diff
@@ -0,0 +1,83 @@
+import Command, {TCommandExampleList} from '../../Command.js';
+import {REQUEST_ID_SIZE, ICommandParameters} from '../../CommandBinaryBuffer.js';
+import {DOWNLINK} from '../../constants/directions.js';
+
+
+/**
+ * IAddMeterProfileParameters command parameters
+ */
+interface IAddMeterProfileParameters extends ICommandParameters {
+    meterProfileId: number
```

Comment:
maybe just `id`?

Reply:
It could not be obvious, then. In the review below, `get meter archive profile,` you proposed to change from meterProfileId to meterId, which is wrong. It is much more precise now.